src/routes.js

```diff
@@ -12,39 +12,19 @@ const Dashboard = Loadable({
   loading: Loading,
 });
 
-const Bitcoin = Loadable({
-  loader: () => import('./views/Bitcoin/Bitcoin'),
-  loading: Loading,
-});
-
-const Ethereum = Loadable({
-  loader: () => import('./views/Ethereum/Ethereum'),
-  loading: Loading,
-});
-
-const Litecoin = Loadable({
-  loader: () => import('./views/Litecoin/Litecoin'),
-  loading: Loading,
-});
-
-const Monero = Loadable({
-  loader: () => import('./views/Monero/Monero'),
-  loading: Loading,
-});
-
-const XRP = Loadable({
-  loader: () => import('./views/XRP/XRP'),
+const CryptoDetails = Loadable({
+  loader: () => import('./views/CryptoDetails/CryptoDetails'),
   loading: Loading,
 });
 
 const routes = [
   { path: '/', exact: true, name: 'Home', component: DefaultLayout },
   { path: '/dashboard', name: 'Dashboard', component: Dashboard },
-  { path: '/bitcoin', name: 'Bitcoin', component: Bitcoin },
-  { path: '/ethereum', name: 'Ethereum', component: Ethereum },
-  { path: '/litecoin', name: 'Litecoin', component: Litecoin },
-  { path: '/monero', name: 'Monero', component: Monero },
-  { path: '/ripple', name: 'XRP', component: XRP },
+  { path: '/bitcoin', name: 'Bitcoin', symbol: 'BTC', twitterScreenname: 'bitcoin', component: CryptoDetails },
```

Comment:
I noticed we were really only passing down about 3 unique values. The name, symbol and twitter screenname so this should remove a lot of the redundant files. Biggest concern with the previous pattern is how that wouldn't scale well with the duplication a couple hundred currencies would have.